docs/dev_guide/diagnostic_ref/dpct1059.rst

```diff
@@ -20,51 +20,72 @@ generates code with unsupported image format, which corresponds to the original
 code. You can fix the resulting code by changing the image format. Note: suggested
 workaround may impact code performance.
 
-For example, the following migrated SYCL code:
+For example, this original CUDA\* code:
 
 .. code-block:: cpp
    :linenos:
 
-     // migrated SYCL code, which is using unsupported image format:
-  
-     dpct::image_wrapper<cl::sycl::uint2, 2> tex; // 2-channel image is not supported
-  
-     void test_image(dpct::image_accessor_ext<cl::sycl::uint2, 2> acc) {
-       cl::sycl::uint2 tex_data;
-       tex_data = acc.read(0, 0);
-     }
-     int main() {
-       ... dpct::get_default_queue().submit([&](cl::sycl::handler &cgh) {
-         ... auto acc = tex.get_access(cgh);
-         auto smpl = tex.get_sampler();
-         ... cgh.single_task<class dpct_single_kernel>([=] {
-           test_image(dpct::image_accessor_ext<cl::sycl::uint2, 2>(smpl, acc));
-         });
-       });
-       ...
-     }
-
-is manually adjusted to:
+   static texture<uint2, 2> tex;

```

Comment:
maybe also add the pseudo code to load 2 channel data into 4 channel memory.

Reply:
done